apps/token-manager/app/src/components/HolderRow.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  TableRow,
+  TableCell,
+  ContextMenu,
+  ContextMenuItem,
+  IconAdd,
+} from '@aragon/ui'
+
+class HolderRow extends React.Component {
+  static defaultProps = {
+    name: '',
+    balance: 0,
+    groupMode: false,
+  }
+  static ActionLabel = styled.span`
+    margin-left: 15px;
+  `
```

Comment:
Do you prefer this to having an outside component (or was there another advantage to adding it inside the component)?

This seems more complicated to me, especially with the `Self` reference.

Reply:
Yes I was trying this, I was waiting for your feedback ha ha 😆

I’ll go back to the previous style.

The idea was to move everything inside the class, to make it clearer when we have multiple components in a single file (`HolderRow` was in `screens/Holders.js`, then I moved it).

But I agree it’s not as readable, and it’s probably a good thing to make multiple component files look bad, so we are pushed to use a single component per file.

Reply:
> Make it clearer when we have multiple components in a single file

I like solving this by just moving out those multiple components into other files :).